src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay/manage-session-replay-modify-capabilities.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: "Manage session replay & modify capabilities"
+metaDescription: "Manage session replay & modify capabilities depending on the role"
+freshnessValidatedDate: 2024-12-19
+---
+
+Manage session replay & modify capabilities provides instructions on utilizing the new user management access functionality, enabling Admins to control who can modify Session replay settings by customizing user roles and access.
+
+## Delete all product admin role [#modify-all-admin-role]
+
+<Steps>
+
+  <Step>
+    Go to <DNT>**Administration > Access Management > Groups**</DNT>.
+  </Step>
+
+  <Step>
+    Click the ellipsis **(...)** icon and select <DNT>**Manage Group Access**</DNT>.
+  </Step>
+
+  <Step>
+    Expand the **Members** section, select the required user(s).
+  </Step>
+
+  <Step>
+    Click **Remove users from group**.
+  </Step>
+
+  <Step>
+    Expand the **Account access** section, click the ellipsis **(...)** icon.
+  </Step> 
+
+  <Step>
+     Select **Delete role**.
+  </Step>
+
+</Steps>
+
+## Create a custom role for session replay read access [#create-custom-role]
+
+<Steps>
+
+  <Step>
+    Go to <DNT>**Administration > Access Management > Roles**</DNT>.
+  </Step>
+
+  <Step>
+    Click <DNT>**Add New Custom Roles**</DNT> and enter a custom role name.
+  </Step>
+
+  <Step>
+    In the <DNT>**Browser**</DNT> section, select the <DNT>**View**</DNT> checkbox for <DNT>**Session replays**</DNT>. Ensure the <DNT>**Modify**</DNT> checkbox is not selected.
+
+   <img
+     title="Browser Role ID"
+     alt="A screenshot of Browser Role ID"
+     src="/images/browser-role-id.webp"
+   />
+  </Step>
+
+  <Step>
+    Click <DNT>**Save**</DNT>.
+  </Step>
+
+  <Step>  
+    Select the <DNT>**Groups**</DNT> tab and click <DNT>**Create new group**</DNT>.
+  </Step>
+
+  <Step>
+    Enter a <DNT>**Group name**</DNT> and click <DNT>**Create group**</DNT>.
+  </Step>
+
+  <Step>
+    Expand the <DNT>**Members**</DNT> section, select the user(s) from the **Select a user** drop-down list, and click **Add user to group**.
+  </Step>
+
+  <Step>
+    Expand the **Account acess** section and in the **Add account roles to this group** select the following:
+    
+    * **Select a role**
+    * **Select an account**
+  </Step>
+
+  <Step>
+    Click **Add role**.
+  </Step>
+
+</Steps>
+
+## Grant session replay modify capability [#grant-sr-modify-capability]
+
+<Steps>
+
+  <Step>
+    Go to <DNT>**Administration > Access Management > Roles**</DNT>.
+  </Step>
+
+  <Step>
+    In the <DNT>**Custom role**</DNT> section, click the ellipsis **(...)** icon and select <DNT>**Manage role**</DNT>.
+  </Step>
+
+  <Step>
+    Click the <DNT>**Edit**</DNT> icon and go to the <DNT>**Browser**</DNT> section .
+  </Step>
+
+  <Step>
+    Select the <DNT>**Modify**</DNT> checkbox for <DNT>**Session replays**</DNT>.
```

Comment:
Do you mean "In the Session replays section, select...

Reply:
Yes, I have updated the sentence